waspc/test/Generator/MockWriteableMonad.hs

```diff
@@ -36,12 +37,15 @@ defaultMockConfig =
 getMockLogs :: MockWriteableMonad a -> MockWriteableMonadConfig -> MockWriteableMonadLogs
 getMockLogs mock config = fst $ execState (unMockWriteableMonad mock) (emptyLogs, config)
   where
-    emptyLogs = MockWriteableMonadLogs [] [] [] [] [] [] []
+    emptyLogs = MockWriteableMonadLogs [] [] [] [] [] [] [] []
```

Comment:
Ah, you got to see the MockWriteableMonad :D!

It works but it is very boilerplatish. I did this a year or more ago. These days I would look for more advanced solution, something that uses Template Haskell, but at that time I wasn't so brave yet. I hope we refactor it at some point and remove most of this boilerplate.